pkgs/tools/text/ov/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "ov";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "noborus";
+    repo = "ov";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-vBPhCSor3wGCawz+097Lw29xgW6z5fV5PAMAq7TBiNM=";
+  };
+
+  vendorHash = "sha256-y3oSL1W2cjt6oUVbglHhun3XNCidqb7LTXtoA25+mpo=";
+
+  ldflags = [
+    "-X main.Version=v${version}"
+    "-X main.Revision=${src.rev}"
+  ];
+
+  excludedPackages = [
+    "exec"
+    "search"
+    "simple"
+    "tail"
+  ];
+
+  meta = with lib; {
+    description = "Feature-rich terminal-based text viewer";
+    homepage = "https://noborus.github.io/ov";
+    license = licenses.mit;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
`platforms.unix` is the same AFAIK

Reply:
Fair point, we do not have a lot of Cygwin, *BSD, Solaris testers so I did not expect this to be a problem, we can let it as-is.

Reply:
`buildGoModule` has default `platforms`:
https://github.com/NixOS/nixpkgs/blob/245eff71712fa7e157810c9278ccf4c98ced842e/pkgs/build-support/go/module.nix#L306